README.md

```diff
@@ -11,7 +11,7 @@ The philosophy of this project is from something simple then be able to expand t
 To install the tool, clone this repository and run `cargo install --path .`, it should work ! (This solution is by waiting pushing real releases on this Github repository)
  
 - Create a simple binary app : `amargo new <my_app>`
-- Create a dynamic library : `amargo new <my_lib> --dylib`
+- Create a dynamic library : `amargo new <my_lib> -- dynamic`
```

Comment:
Should be `--dynamic` without the space, I'd like to reserve the `--<Args...>` for other things

Reply:
Yeah I saw i was wrong too